### PR TITLE
Bump rancher-operator to release

### PR DESCRIPTION
**Pull request by thedadams**
Issue:
https://github.com/rancher/rancher/issues/30888

### CODE REVIEW

File: packages/rancher-operator-crd/package.yaml [hunk 1]
```diff
@@ -1,3 +1,3 @@
-url: https://github.com/rancher/rancher-operator/releases/download/v0.1.3-rc7/rancher-operator-crd-0.1.3-rc7.tgz
-packageVersion: 00
-releaseCandidateVersion: 07
+url: https://github.com/rancher/rancher-operator/releases/download/v0.1.3/rancher-operator-crd-0.1.3.tgz
+packageVersion: 01
```
Comment (cbron) [inline packages/rancher-operator-crd/package.yaml:2]: I would do packageVersion 00

File: packages/rancher-operator-crd/package.yaml [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
+releaseCandidateVersion: 00
```
Comment (cbron) [inline packages/rancher-operator-crd/package.yaml:3]: We just discovered we have to do +1 on previous RC for this project. See: https://github.com/rancher/charts/pull/1016/files#diff-e92b26bfc5708ae53b7059830653831cad111e8a2b5e21df7126172b654a3af6R3

Comment (thedadams) [reply packages/rancher-operator-crd/package.yaml]: Done

**Review by cbron — APPROVED** [review]
need to pull in upstream, then lg

**Review by cbron — APPROVED** [review]

**Pull request merged**